client/landing/stepper/declarative-flow/internals/steps-repository/importer/hooks/use-step-navigator.ts

```diff
@@ -45,10 +45,10 @@ export function useStepNavigator(
 		navigator( getWordpressImportContentOnlyUrl() );
 	}
 
-	function goToSiteViewPage() {
+	function goToWpAdmin() {
 		navigation.submit?.( {
 			type: 'redirect',
-			url: `/view/${ siteId || siteSlug || '' }`,
+			url: `/home/${ siteId || siteSlug || '' }`,
```

Comment:
That doesn't really go to `/wp-admin` though?

Reply:
Do we have a link that goes to the right "site admin" depending on the site slug?

Reply:
Last time I checked, we didn't have a utility function, but you can see this used across A4A:

```
`${ site.url_with_scheme }/wp-admin`
```

Why do we want to go to admin here, though? Wasn't this previously previewing the site? If so, shouldn't we just open a new window that opens the live site for previewing?

Reply:
That makes sense 👍 One could argue previewing the site was also odd in the first place, and after import you'd want to see the result from an admin's perspective.